config/src/config/state_sync_config.rs

```diff
@@ -180,7 +180,7 @@ impl Default for AptosDataClientConfig {
         Self {
             max_num_in_flight_priority_polls: 10,
             max_num_in_flight_regular_polls: 10,
-            response_timeout_ms: 10000,
+            response_timeout_ms: 50000, // 50 seconds
```

Comment:
I'm not sure I understand fully the implications of this, e.g. will it invite DDoS attacks?

Reply:
No, these are the timeouts for RPC requests that we send. So, in the worst case a malicious peer can just not respond and we'll wait an extra 40 seconds before moving on. But, they would have to be malicious and if they were there's much worse things they could do not related to timeouts :D